harvest_eads/create_json_from_xml_xpath.py

```diff
@@ -33,6 +34,8 @@ def __init__(self, config, json_control):
         self.temporary_local_path = "/tmp"
         self.sequences_within_parent = {}
         self.output_csv_class = OutputCsv(self.config["csv-field-names"])
+        self.event['local'] = event.get("local", False)
```

Comment:
I'd recommend just storing local as a class variable and removing line 28 all together.
```suggestion
self.local_run = event.get("local", False)
```